KubeArmor/core/containerPolicy_fuzz_test.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2024 Authors of KubeArmor
+package core
+
+import (
+	"context"
+	"testing"
+
+	"github.com/kubearmor/KubeArmor/KubeArmor/policy"
+	pb "github.com/kubearmor/KubeArmor/protobuf"
+)
+
+func FuzzContainerPolicy(f *testing.F) {
+	initialData := &pb.Policy{
+		Policy: []byte(`
```

Comment:
Let's use varied kinds of policy including file, network as seed values. https://github.com/kubearmor/KubeArmor/tree/main/tests/k8s_env/ksp/multiubuntu